src/main/webapp/feed.html

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="/css/navbar.css">
 
     <script src="/js/messages-loader.js"></script>
+    <meta http-equiv = "refresh" content = "8; url = /feed.html" />
```

Comment:
Let's not do a continuous refresh. Let's only refresh once after 4.5s, ONLY if they came from the maps.html page. One way we can do that is that the maps.html page redirects users to something like feed.html?refresh=true and so whenever the url param 'refresh' is true, we refresh ONCE after 4.5s.

Reply:
oh okay, thanks for the tip! I'll try that

Reply:
Can I just do reload once on feed.html page itself? Because now backend implementation is changed to:
```
async function handleSubmitButtonClick(key, text) {
  await postMessage(key, text);
  window.location.href = "/feed.html?cinemaKey=" + key;
}
```
so I am abit unsure of how to do reload ONLY if they came from maps.html.

Reply:
uh I think I might need more advice on how to implement this

Reply:
Right now the backend implementation of redirecting from maps page is changed to this:
```
async function handleSubmitButtonClick(key, text) {
  await postMessage(key, text);
  window.location.href = "/feed.html?cinemaKey=" + key;
}
```

Can I ask for advice on how to do the single refresh?

Reply:
Okay I managed to do single refresh after 4.5 secs, but its called from feed page instead of maps page. Can we use this version? @potay